README.md

```diff
@@ -482,22 +540,24 @@ file as follows:
 Then in your activity, declare the method that will start this activity as follows:
 
 ```kotlin
-private fun showAuthorizingPaymentForm() {
+private fun startAuthoringPaymentActivity() {
     Intent(this, AuthorizingPaymentActivity::class.java).run {
             putExtra(EXTRA_AUTHORIZED_URLSTRING, authorizeUrl)
             putExtra(EXTRA_EXPECTED_RETURN_URLSTRING_PATTERNS, arrayOf(returnUrl))
+            putExtra(EXTRA_UI_CUSTOMIZATION, uiCustomization)
             putExtra(
                 EXTRA_THREE_DS_REQUESTOR_APP_URL,
                 "sampleapp://omise.co/authorize_return"
             )
-            startActivityForResult(this, AUTHORIZING_PAYMENT_REQUEST_CODE)
+            authorizingPaymentLauncher.launch(this)
         }
 }
 ```
 
-Replace the string `AUTHORIZED_URL` with the authorized URL that comes with the created charge and the array of string `EXPECTED_URL_PATTERNS` with the expected pattern of redirected URLs array.
-
+Replace the string `EXTRA_AUTHORIZED_URLSTRING` with the authorized URL that comes with the created charge and the array of string `EXTRA_EXPECTED_RETURN_URLSTRING_PATTERNS` with the expected pattern of redirected URLs array.
+The `EXTRA_UI_CUSTOMIZATION` parameter is used to customize the UI in the built in 3DS in app flow in the SDK and during the challenge flow.
```

Comment:
built-in

Reply:
better to say: The `EXTRA_UI_CUSTOMIZATION` parameter is used to customize UI of the built-in 3DS SDK during the 3DS challenge flow.